projects/plugins/crm/js/ZeroBSCRM.admin.invoicebuilder.js

```diff
@@ -1582,10 +1582,13 @@ function zbscrm_JS_calc_amount_due() {
 			v = parseFloat( v ) * multiplier;
 
 			// if v is a negative number and also a Credit or Refund, add to amount_due, otherwise, deduct.
-			if ( v < 0 && ( label.includes( 'Refund' ) || label.includes( 'Credit' ) ) ) {
-				amount_due += v;
-			} else {
-				amount_due -= v;
+			// if ( v < 0 && ( label.includes( 'Refund' ) || label.includes( 'Credit' ) ) ) {
+			// 	amount_due += v;
+			// } else {
+			// 	amount_due -= v;
+			// }
```

Comment:
We have to remove this commented code.

Reply:
Yes, thanks for testing! It looks like the logic wasn't fully completed, so this section should be (at quick glance):
```
if (  label.includes( 'Refund' ) || label.includes( 'Credit' ) ) {
amount_due -= Math.abs( v );
} else {
amount_due -= v;
}
```
Which would fix the sales amount issue above too.